ios/CameraView+Focus.swift

```diff
@@ -18,7 +18,8 @@ extension CameraView {
         throw CameraError.device(DeviceError.focusNotSupported)
       }
 
-      let normalizedPoint = self.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)
+//      let normalizedPoint = self.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)
+      let normalizedPoint = CGPoint.zero
```

Comment:
In this case, the focus on click stops working, what can be done to make the focus on click work with the frameProcessor?